skyportal/handlers/public/release.py

```diff
@@ -0,0 +1,89 @@
+import sqlalchemy as sa
+
+from baselayer.app.models import DBSession
+from ...models.public_pages.public_source_page import PublicSourcePage
+from ...models.public_pages.public_release import PublicRelease
+
+from ..base import BaseHandler
+
+
+class ReleaseHandler(BaseHandler):
+    def get(self, link_name=None):
+        """
+        ---
+        single:
+            description: Display the page with all sources and descriptions for a given public release
+            tags:
+              - release
+            parameters:
+                - in: path
+                  name: link_name
+                  required: true
+                  schema:
+                    type: string
+                  description: The link name of the public release to display
+            responses:
+                200:
+                  content:
+                    text/html:
+                      schema:
+                        type: string
+                        description: The HTML content of the selected release page
+                404:
+                  content:
+                    application/json:
+                      schema: Error
+        multiple:
+            description: Display the page with all the public releases
+            tags:
+              - release
+            responses:
+                200:
+                  content:
+                    text/html:
+                      schema:
+                        type: string
+                        description: The HTML content of the page with a list of all public releases
+        """
```

Comment:
As discussed, with some good cache invalidation we could consider caching those in the future. Please open an issue once merged. Let's not do that here.